tests/test_dt.py

```diff
@@ -166,3 +166,20 @@ def test_total_minutes(timedeltas: timedelta) -> None:
     assert result_pda == result_pl
     assert result_pdn == result_pl
     assert result_pdns == result_pl
+
+
+@pytest.mark.parametrize("constructor", [pd.DataFrame, pl.DataFrame])
+@pytest.mark.parametrize(
+    "fmt", ["%Y-%m-%d", "%Y-%m-%d %H:%M:%S", "%Y/%m/%d %H:%M:%S", "%G-W%V-%u", "%G-W%V"]
```

Comment:
Polars has a more exhaustive suite. Let me know if you want to replicate or this is good enough.

P.s. to replicate, it would be good to have `.item()`

Reply:
`item` would be good 👍 I think this is fine anyway